Header header
string name
float64 ar_time
bool status

string name1
int32 id1
bool status1
float64 wait1
float64 x1
float64 y1
float64 theta1

string name2
int32 id2
bool status2
float64 wait2
float64 x2
float64 y2
float64 theta2
